applications/ShapeOptimizationApplication/python_scripts/model_part_controller_factory.py

```diff
@@ -35,8 +35,9 @@ def __init__(self, model_settings, model):
             },
             "design_surface_sub_model_part_name" : "DESIGN_SURFACE_NAME",
             "damping" : {
-                "apply_damping"   : false,
-                "damping_regions" : []
+                "apply_damping"      : false,
+                "max_neighbor_nodes" : 10000,
```

Comment:
Have you tried what happens it the limit is reached? I guess it will result in a kink in the shape. Unlike the filtering the damping is not so forgiving when it comes to cutting the used filter function...

Reply:
Ok i just see that it was already there but hardcoded...